apps/src/aichat/chatEventLogger.ts

```diff
@@ -45,11 +43,14 @@ export default class ChatEventLogger {
         this.sendingInProgress = true;
         try {
           await postLogChatEvent(chatEvent, aichatContext);
-          this.sendingInProgress = false;
         } catch (error) {
-          Lab2Registry.getInstance()
-            .getMetricsReporter()
-            .logError('Error in aichat event logging request', error as Error);
+          console.error(
```

Comment:
I was thinking more something like what we do here, where we handle the 403 error (maybe we just do nothing for a 403), but we still log an error to CloudWatch if it's unhandled. I think as written this would swallow errors of any kind (even unexpected ones, which we do want to log to CloudWatch).

https://github.com/code-dot-org/code-dot-org/blob/8cafe0b169b416b75b3f05b1e2f6ca87ffd7639d/apps/src/aichat/redux/aichatRedux.ts#L458

Reply:
Oh gotcha - that makes sense. Will update!